openwillis-face/src/openwillis/face/face_landmark.py

```diff
@@ -11,8 +11,7 @@
 import mediapipe as mp
 from protobuf_to_dict import protobuf_to_dict
 
-from openwillis.measures.video.util.crop_utils import crop_with_padding_and_center
-from openwillis.measures.video.util.speaking_utils import get_speaking_probabilities, split_speaking_df, get_summary
+from .util import crop_with_padding_and_center, get_speaking_probabilities, split_speaking_df, get_summary
```

Comment:
why are we not importing from base location? Will it work if we import functions in WillisMeasure; I am not sure so checking with you.